lib.nix

```
@@ -98,6 +98,34 @@ let
         type = flake-parts-lib.mkPerSystemType module;
       };
 
+    # Helper function for defining a per-system option that
+    # gets transposed by the usual flake system logic to a
+    # top-level flake attribute.
+    mkPerSystemModule = { name, option, file }: {
```

```suggestion
    mkTransposedPerSystemModule = { name, option, file }: {
```

The transposition is quite relevant, as a perSystem module could be just one of those submodules.

It's also about a single attribute, but representing that seems to get too long.

---

Done.